sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -140,61 +142,51 @@ The following section provides several JavaScript code snippets illustrating com
 * [Recognize forms using a custom model](#recognize-forms-using-a-custom-model)
 * [Listing all models](#listing-all-models)
 
-### Recognize receipts
+### Recognize Forms Using a Custom Model
```

Comment:
since we are re-ordering samples, should we reorder the anchor links under the Examples header as well?